libraries/AP_HAL/UARTDriver.h

```diff
@@ -143,7 +143,7 @@ class AP_HAL::UARTDriver : public AP_HAL::BetterStream {
     virtual uint64_t receive_time_constraint_us(uint16_t nbytes) { return 0; }
 
     virtual uint32_t bw_in_bytes_per_second() const {
-        return 5760;
+        return 57600;
```

Comment:
By making it one line, it is the same as any other description.
Only this definition does not appear to need to be on more than one line.